### DISCUSSION TITLE
Selection hitbox for trains, trucks, etc. should be the size of the Storages label

### DISCUSSION THREAD

**Discussion by bigyihsuan** [voxeltycoon/issues#478] [General]
The selection hitbox for vehicles are only the size of the vehicle, and not the Storages label.

This is most notable with trying to select a truck, with Storages on, and zoomed all the way out. You can click on the Storages label, but it won't do anything unless you click exactly where the trick is.

**Comment by katepolak** [comment]
Please update the body of the issue to use the suggestion template.
